packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -466,13 +466,13 @@ export const style = createTheme({
         isFocusVisible: weirdColorToken('negative-background-color-key-focus'),
         isPressed: weirdColorToken('negative-background-color-down')
       },
+      // TODO: deprecated in favor of negative-subtle. What is our process around deprecating colors here?
```

Comment:
How do we wanna handle deprecating colors? I can delete it but dunno if we wanna put it in a deprecation period just in case?